common/net.c

```diff
@@ -333,11 +333,14 @@ int udp_connect(int local_port,char *remote_host,int remote_port)
       }
 
       /* try to connect to remote host */
-      setsockopt(sck, SOL_SOCKET, SO_REUSEADDR, &yes, sizeof(yes));
+      error = setsockopt(sck, SOL_SOCKET, SO_REUSEADDR, &yes, sizeof(yes));
       if (!bind(sck,(struct sockaddr *)&st,res->ai_addrlen) &&
           !connect(sck,res->ai_addr,res->ai_addrlen))
          break;
 
+      if (error) {
+         fprintf(stderr,"Warning: udp_connect: SO_REUSEADDR is not set. The same address-port combination can be retried after the TIME_WAIT state expires.");
```

Comment:
According to the man page, setsockopt can return different errors. Should we use a more generic perror() instead?

Reply:
I can also include the system error message but the question remains, is that message clear enough for network people to understand what is going on?

Reply:
You can say "TCP TIME_WAIT state expires." to be more explicit and add the system error message. I think that should be more than enough. Thanks :+1: